analyzers/src/SonarAnalyzer.CSharp/Extensions/SyntaxNodeExtensions.cs

```diff
@@ -199,6 +198,7 @@ node switch
                 PointerTypeSyntax { ElementType: { } elementType } => GetIdentifier(elementType),
                 PredefinedTypeSyntax { Keyword: var keyword } => keyword,
                 QualifiedNameSyntax { Right.Identifier: var identifier } => identifier,
+                SimpleBaseTypeSyntax { Type: { } type } => GetIdentifier(type),
```

Comment:
As long as we do not use version of Roslyn with nullable annotations, we should always assume null as a possible value. Otherwise, we a re always in the "not sure" camp and have to consult `Syntax.xml` or any other kind of documentation. Let's just play "Better safe than sorry".